.github/CODEOWNERS

```diff
@@ -1,7 +1,21 @@
-# Lines starting with '#' are comments.
-# Each line is a file pattern followed by one or more owners.
+# cumulocity-examples CODEOWNERS
+# Keep the directory order alphabetical and always list subdirectories after their parents
+# Keep in mind that cumulocity-examples is open source so everything in this file is public
 
-# Either @username or email.address@domain.com can be used
-
-# These owners will be the default owners for everything in the repo.
-* philipp.emmel@softwareag.com
+/* @martin-sag
+.*/** @martin-sag
+/device-integrations @l2p-
+/hello-world-microservice @mgrumann
```

Comment:
Please remove me as code owner for all these examples. These samples are no longer necessary. At least customers are pointed to https://github.com/SoftwareAG/cumulocity-microservice-templates which are a) document and b) actively maintained. I would propose deleting the code if nobody else would like to take the responsibility for it.

Reply:
These examples are referenced in public docs: https://cumulocity.com/docs/microservice-sdk/java/#introduction. If they should no longer be used I suggest you take ownership, change the docs, and replace the segments in this repo with links to their replacement.

Reply:
Sorry guys, I don't think it is a good idea to remove the code of the examples microservice. We need to have a running example somewhere. Imho I believe this microservice should be managed by the team "responsible" for the SDK.

Reply:
that is exactly what is happening here. ownership goes to the respective person/team and they will deal with the future changes including removal or moving to a different place.